Makefile

```diff
@@ -55,7 +53,24 @@ ifeq ($(TARGET),aarch64-apple-darwin)
 	PLATFORM_OPENSSL_CONFIG += darwin64-arm64-cc
 	CFLAGS = -arch arm64 -mmacosx-version-min=$(MACOSX_DEPLOYMENT_TARGET)
 	LDFLAGS = -arch arm64 -mmacosx-version-min=$(MACOSX_DEPLOYMENT_TARGET)
-	PLATFORM_OPENVPN_CONFIG = --target=aarch64-apple-darwin --host=aarch64-apple-darwin
+	PLATFORM_OPENVPN_CONFIG = --host=aarch64-apple-darwin
+endif
+
+ifeq ($(TARGET),aarch64-unknown-linux-gnu)
+	# ARM doesn't support 'mcmodel=large'
+	LIBNFTNL_CFLAGS = "-g -O2"
```

Comment:
Nit. This is not something you added, but that we can improve. This feels backwards. The comment and the code does not connect. One has to know that `mcmodel=large` **is already in the variable** when the code comes here. It's not obvious that the code removes a flag. In all other places we instead define the base flags and optionally **add** flags. Can we do something similar to this instead?

```
LIBNFTNL_CFLAGS = "-g -O2"
...
# ARM doesn't support 'mcmodel=large'
ifneq ($(TARGET),aarch64-unknown-linux-gnu)
    LIBNFTNL_CFLAGS += -mcmodel=large
endif
```